src/sapG_fmt_plug.c

```diff
@@ -458,8 +458,7 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 #if defined(_OPENMP)
 #pragma omp parallel for
 #endif
-	for (t = 0; t < (count-1)/(NBKEYS)+1; t++)
-	{
+	for (t = 0; t < (count-1)/(NBKEYS)+1; t++) {
```

Comment:
We generally use signed int for the index variable in OpenMP `for` loops, because support for unsigned requires OpenMP 3.0 spec support from the compiler, IIRC. Maybe we depend on that spec (or higher?) anyway, but at least the core tree tries to target OpenMP 2.5+.

Reply:
This should be fixed now.

```
@@ -454,12 +448,11 @@ static int crypt_all(int *pcount, struct db_salt *salt)
 #if SIMD_COEF_32
 #define ti (t*NBKEYS+index)
 
-       unsigned t;
+       int t;
 #if defined(_OPENMP)
```